### PR TITLE
Convert "generate-stackbrew-library.sh" to output the new 2822-based format

### PR DESCRIPTION
**Pull request by mattrobenolt**
@tianon @yosifkit 

This is mostly a copy/paste from what you're doing over there on other images.

The output at HEAD is:

```
Maintainers: Matt Robenolt <matt@getsentry.com> (@mattrobenolt)
GitRepo: https://github.com/getsentry/docker-sentry.git

Tags: 8.4.1-onbuild, 8.4-onbuild
GitCommit: 80218c227b188fad17575040421d600303c5f7bf
Directory: 8.4/onbuild

Tags: 8.4.1, 8.4
GitCommit: 80218c227b188fad17575040421d600303c5f7bf
Directory: 8.4

Tags: 8.5.1-onbuild, 8.5-onbuild, 8-onbuild, onbuild
GitCommit: ce5121a71f55c2fb0659f552e361b1174c85bccf
Directory: 8.5/onbuild

Tags: 8.5.1, 8.5, 8, latest
GitCommit: ce5121a71f55c2fb0659f552e361b1174c85bccf
Directory: 8.5
```

This _seems_ ok.

### CODE REVIEW

File: generate-stackbrew-library.sh [hunk 1]
```diff
@@ -1,44 +1,86 @@
...
+			Directory: $version/$variant
+		EOE
 	done
+
+	echo
+	cat <<-EOE
+		Tags: $(join ', ' "${versionAliases[@]}")
+		GitCommit: $commit
+		Directory: $version
+	EOE
```
Comment (tianon) [inline generate-stackbrew-library.sh]: If you move this section up before `for variant in onbuild; do`, then your normal `latest` tag will be listed first in "supported tags and dockerfile links" on the Hub description, but it's up to you on what order you'd like there :+1:

**Comment by tianon** [comment]
LGTM, just one little comment that depends on your own preference :+1:

File: generate-stackbrew-library.sh (hunk 1 above)
Comment (mattrobenolt) [inline generate-stackbrew-library.sh]: I'm not sure what you mean. Can you clarify?

Comment (mattrobenolt) [inline generate-stackbrew-library.sh]: Oh, nevermind. I see what you mean.

Comment (tianon) [inline generate-stackbrew-library.sh]: :+1:

**Pull request merged**